### PR TITLE
Bug 1557209, Updated openshift_master_oauth_templates

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1557209

### CODE REVIEW

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -887,7 +886,6 @@ xref:../install_config/install/advanced_install.adoc#advanced-install-configurin
 
 # Configure a custom login template in the master config
 # See: https://docs.openshift.com/enterprise/latest/install_config/web_console_customization.html#customizing-the-login-page
```
Comment (spadgett) [inline install_config/web_console_customization.adoc]: The two lines of comments above should be removed as well.

**Comment by spadgett** [comment]
It's not clear to me why this is deprecated. Is it only the installer var deprecated or the entire feature?

cc @simo5 @enj

**Comment by enj** [comment]
> It's not clear to me why this is deprecated. Is it only the installer var deprecated or the entire feature?

@spadgett looks like it was moved around in Ansible: https://github.com/openshift/openshift-ansible/pull/6522

Nothing has changed in OpenShift; the login / provider / error templates continue to work as before.

**Comment by ahardin-rh** [comment]
Extra lines are now removed. Standing by for guidance on an updated example.

**Comment by ahardin-rh** [comment]
@michaelgugino Can you please help with an example?

**Comment by michaelgugino** [comment]
@ahardin-rh 

https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example#L97-L98

We try to document most user-accessible variables in hosts.example.

**Comment by spadgett** [comment]
@michaelgugino Can you use that syntax in an inventory file? I didn't think that worked when using INI.

**Comment by michaelgugino** [comment]
@spadgett that's a good point.  I'll get that fixed, and link the PR here.

**Comment by michaelgugino** [comment]
@ahardin-rh @spadgett Updated example here: https://github.com/openshift/openshift-ansible/pull/8460

**Comment by ahardin-rh** [comment]
@spadgett @michaelgugino Thanks! Updated PTAL

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by ahardin-rh** [comment]
QE verified in the BZ

**Pull request merged**